data-collection/src/main/java/refactoringml/App.java

```diff
@@ -214,7 +216,7 @@ private void processCommit(RevCommit currentCommit, GitHistoryRefactoringMiner m
 
 			//collect the process metrics for the current commit
 			processMetrics.collectMetrics(currentCommit, superCommitMetaData, allRefactoringCommits);
-			db.commit();
+			db.commit(commitHash);
```

Comment:
I'd not pass this `commitHash` variable to the `commit` method, as it's only used for logging. I'd time the time it takes from here, e.g. 

```
startTimer
db.commit()
log.debug(time it took)
```

So that all the ugliness stays here in the App and doesn't propagate to the db class